lib/openstudio/extension/runner_config.rb

```diff
@@ -60,11 +76,18 @@ def self.default_config
       #
       #  @param [String] dirname Directory where runner.conf file is stored, typically the root of the extension.
       def self.init(dirname)
-        File.open(File.join(dirname, FILENAME), 'w') do |f|
-          f << JSON.pretty_generate(default_config)
+        runner_conf_file = File.join(dirname, FILENAME)
+        if File.exist?(runner_conf_file)
+          puts "runner.conf already exists, will be overwritten, previous config:"
+          puts File.read(runner_conf_file)
+        end
+
+        config = default_config(dirname)
+        File.open(runner_conf_file, 'w') do |f|
+          f << JSON.pretty_generate(config)
         end
 
-        return default_config
+        return config
```

Comment:
Warn when overwritting runner.conf and print previous so one can revert manually if needed

Reply:
I'd suggest writing out the previous config to a file... save to `.backup` or `.previous`